src/jdk.jartool/share/classes/sun/tools/jar/Main.java

```diff
@@ -69,14 +69,13 @@
 import jdk.internal.module.ModuleResolution;
 import jdk.internal.module.ModuleTarget;
 import jdk.internal.opt.CommandLine;
-import jdk.internal.util.jar.JarIndex;
+
 import java.time.LocalDateTime;
```

Comment:
This can move up to the imports of the other java.* classes.

Reply:
Fixed.